src/fut/mod.rs

```diff
@@ -44,7 +44,38 @@ use actor::Actor;
 /// Trait for types which are a placeholder of a value that may become
 /// available at some later point in time.
 ///
-/// This is similar to `futures::Future` trait, except it works with `Actor`
+/// `ActorFuture` is very similar to a regular `Future`, except for the fact that `map`ing over it
+/// works with closures accepting the relevant actor and its context.
+///
+/// `ActorFuture` is required in cases where future processing requires access to the actor ot its context -
```

Comment:
Rather than saying `required` it is better to say that there are use cases such as: allowing access actor/context from within future, waiting for future within actor's context.